non-critical-infra/modules/mailserver/mailing-lists.nix

```
@@ -169,6 +169,13 @@
       ];
     };
 
+    "securitytracker-noreply@nixos.org" = {
```

```suggestion
    "noreply-securitytracker@nixos.org" = {
```

That would be my preference, so that people actually notice the noreply part. I do the same for DMARC and TLSRPT in SNM.